public/views/index.html

```diff
@@ -11,15 +11,15 @@
         <li><a ng-click="goTo('how-to-play')">HOW TO PLAY</a></li>
         <li><a ng-click="goTo('charity')">CHARITY</a></li>
         <li ng-show="showOptions"><a href="#!signup">SIGN UP</a></li>
-        <li ng-show="showOptions"><a href="#!signin">SIGN IN</a></li>
         <li ng-hide="showOptions"><a href="/signout">SIGN OUT</a></li>
+        <li ng-show="showOptions"><button class="waves-effect waves-light btn" data-target='modal1' id='signin'>SIGN IN</button></li>
```

Comment:
For consistency with other nav links use this
`<li ng-show="showOptions" data-target='modal1'><a>SIGN IN</a></li>`